### PR TITLE
Wrapping Up BI Integration

### PR DESCRIPTION
**Pull request by biyeun**
- make sure split columns are actually passed to the schema
- make sure the value in the json output is always a string
- text changes to odata specific views in exports
- track analytics events related to odata interactions

best reviewed commit-by-commit

### CODE REVIEW

File: corehq/apps/export/models/new.py [hunk 1]
```diff
@@ -522,12 +522,11 @@ def get_rows(self, document, row_number, split_columns=False,
...
-                        headers = col.get_headers(split_column=split_columns)
-                        for subval_ind, subval in enumerate(val):
-                            row_data[headers[subval_ind]] = "{}".format(subval)
-                    else:
-                        row_data[col.label] = "{}".format(val)
+                    for ind, header in enumerate(col.get_headers(split_column=split_columns)):
+                        if isinstance(val, list):
+                            row_data[header] = "{}".format(val[ind])
+                        else:
+                            row_data[header] = "{}".format(val)
```
Comment (czue) [inline corehq/apps/export/models/new.py]: i'm a little unclear why this change was necessary. this puts the same value in multiple columns if there were multiple headers but a single value? when does that happen?

**Review by czue — COMMENTED** [review]
not feeling confident enough about the context I have to approve, but no issues on my side.

do the deid changes impact anything user-facing or that is just code cleanup?

File: corehq/apps/export/models/new.py (hunk 1 above)
Comment (biyeun) [reply corehq/apps/export/models/new.py]: if there is a single value there is only a single header. there isn't really an instance where there are multiple headers and one value. this just simplifies the code a bit

Comment (biyeun) [reply corehq/apps/export/models/new.py]: unfortunately, if there is a single value, it is no longer a list...so `val[0]` doesn't work. so perhaps what we should really address is that inconsistency in the exports code!

Comment (czue) [reply corehq/apps/export/models/new.py]: makes sense, thanks. tiny unrelated suggestion you can take or leave is maybe rename `ind` to `index` for readability. `ind` is a little nonstandard and for some reason makes me think of "indicator"

**Review by orangejenny — APPROVED** [review]
